services/upload/service/upload_service.go

```diff
@@ -58,12 +68,18 @@ func UpdateUserResume(id string, file_buffer []byte) error {
 		return errors.New("Resume upload must be a pdf")
 	}
 
-	_, err := uploader.Upload(&s3manager.UploadInput{
-		Bucket:      aws.String(config.S3_BUCKET),
-		Key:         aws.String("resumes/" + id + ".pdf"),
-		Body:        bytes.NewReader(file_buffer),
-		ContentType: &content_type,
-	})
+	var err error
+	if config.IS_PRODUCTION {
+		_, err = uploader.Upload(&s3manager.UploadInput{
+			Bucket:      aws.String(config.S3_BUCKET),
+			Key:         aws.String("resumes/" + id + ".pdf"),
+			Body:        bytes.NewReader(file_buffer),
+			ContentType: &content_type,
+		})
+	} else {
+		os.Mkdir("/tmp/uploads", os.ModePerm)
+		err = ioutil.WriteFile("/tmp/uploads/"+id+".pdf", file_buffer, 0644)
```

Comment:
We have spaces before the `+` everywhere else, so we should have it here as well.